src/app/public/modules/datepicker/datepicker.component.spec.ts

```diff
@@ -50,6 +50,7 @@ import {
 const moment = require('moment');
 
 // #region helpers
+class MockWindowService extends SkyWindowRefService {}
```

Comment:
What is the purpose of this if we aren't adding or changing any methods? Could we just not use the `inject` function to get the actual `SkyWindowRefService` from the injector? In the previous version this made sense as we were overwriting the `getWindow` function but it doesn't appear we are doing that now (unless I'm missing something)

Reply:
Check out the describe block at line `1135`. We have a couple tests that cover locale logic. I made this a class instead of using `inject` because we have more than one tests that's using it. I figured this was cleaner and more reusable if we want more tests to use it in the future.

Reply:
Ok. I don't know that I 100% love mocking for the sake of mocking when spys will work but nothing really to make a big deal about. Personal preference really :) Its fine with me at this point.